src/style.css

```diff
@@ -0,0 +1,103 @@
+@font-face {
+  font-family: "Avenir";
+  src: url(fonts/Avenir-Book.ttf);
+  font-style: normal;
+  font-weight: normal;
+}
+
+@font-face {
+  font-family: "Avenir";
+  src: url(fonts/Avenir-Heavy.ttf);
+  font-weight: bold;
+}
+
+* {
+  box-sizing: border-box;
+}
+
+html {
+  font-family: "Averin", sans-serif;
+}
+
+.search-form {
+  margin: 20px 0;
+}
+
+.image-search {
+  position: absolute;
+}
+
+.big {
+  width: 19px;
+  height: 19px;
+  margin-left: 26px;
+  margin-top: 25px;
+}
+
+.small {
+  width: 11px;
+  height: 11px;
+  margin-top: 15px;
+  margin-left: 13px;
+}
+
+.search-bar__field {
+  box-shadow: 0 1px 8px rgb(61 78 97 / 10%);
+  border: 1px solid #e1e7ed;
+  border-radius: 4px;
+  width: 100%;
+  height: 70px;
+  padding-left: 62px;
+  background: #fff;
+  outline: none;
+  font-size: 16px;
+  font-weight: 300;
+  line-height: 22px;
+  color: #3d4e61;
+}
+
+.search-bar__field:hover {
+  display: flex;
+  align-items: center;
+  background: #fff;
+  border: 1px solid #e1e7ed;
+  box-shadow: 0 3px 8px rgb(61 78 97 / 20%);
+  border-radius: 4px;
```

Comment:
you should change properties like colors, radius, etc. but not display or positioning or border (set only border-color)